content/docs/operating/storage.md

```diff
@@ -253,7 +295,9 @@ that cannot be recovered is moved to a directory called `orphaned`
 you do not need it anymore.
 
 The crash recovery usually takes less than a minute. Should it take much
-longer, consult the log to find out what has gone wrong.
+longer, consult the log to find out what is going on. With increasing number of
+time series in the storage (archived or not), the re-indexing tends to dominate
+the recovery time and can take tens of minutes.
```

Comment:
It also depends on the state of the SSD, how big it's block pool for tiny writes it. That's why SSDs gets the slower the fuller they are. In my enforced crash recovery, the SSDs was running hot continuously for a long time. I guess, the current wording "in extreme cases" catches that.

I only want to avoid more issues filed as "crash recovery took 20m but docs say a few minutes".